src/tests/PhysicsTest.cpp

```diff
@@ -148,888 +148,858 @@ struct PhysicsTest : Cr::TestSuite::Tester {
 
   int sceneID_;
 };  // struct PhysicsTest
+const struct {
+  const char* name;
+  bool enabled;
+} RendererEnabledData[]{{"", true}, {"renderer disabled", false}};
```

Comment:
Might make sense to tell clang-format to ignore this piece of code, this way it's extremely unclear how many test instances are there.